### PR TITLE
[Android] Fix scanned object bounds (coordinates)

### LINKED ISSUE #1156: Customize Barcode scanner view - Android
### Which implementation are you using
RCTCamera 

Is there a way to add a surrounding box around the scanner view. I am looking at the `zxing` examples and see you can achieve something like this 

3524e374-e8f7-11e4-83a1-6d2f75b6a50e

Is there a way to get a similar view in this module. Barcode reader without bounds certainly doesnt tell users as where they should focus there camera on.

Thank you,

### PR DESCRIPTION
**Pull request by raphaelhovsepyan**
Will Fix #1156 #1503

### CODE REVIEW

File: android/src/main/java/com/lwansbrough/RCTCamera/MutableImage.java [hunk 1]
```diff
@@ -167,7 +167,7 @@ private static Bitmap toBitmap(byte[] data) {
     }
 
     public String toBase64(int jpegQualityPercent) {
-        return Base64.encodeToString(toJpeg(currentRepresentation, jpegQualityPercent), Base64.NO_WRAP);
+        return Base64.encodeToString(toJpeg(currentRepresentation, jpegQualityPercent), Base64.DEFAULT);
```
Comment (sibelius) [inline android/src/main/java/com/lwansbrough/RCTCamera/MutableImage.java]: is this a breaking change?

Comment (hayrapetyanabraham) [reply android/src/main/java/com/lwansbrough/RCTCamera/MutableImage.java]: Sorry, I just see I have a conflict, with master branch. Will fix and commit.

**Comment by jgfidelis** [comment]
Looks good, can you try to explain better what are you changing? How did it behave before and how will it behave now?

**Comment by raphaelhovsepyan** [comment]
@jgfidelis So much code changed after our fork, @hayrapetyanabraham  will pull fresh changes and push during next week.

**Comment by fozhao** [comment]
Hey, thanks for trying to getting closer. The current 'bounds' info in the onBarCodeRead callback contains the width/height of the camera canvas and the pattern finder points in the camera canvas coordinates(here 1600 * 1200 on my Pixel 1st-gen). However, on the RN side, it'd be really useful to take some coordinates in the camera preview view's coordinate system.

**Comment by nurchulis** [comment]
Thanks

**Pull request merged**